src/main.js

```diff
@@ -157,3 +178,72 @@ function starFavorite(event) {
   }
   displayCard();
 }
+
+function showStarredIdeas() {
+  ideaCardSection.innerHTML = '';
+  for (var i = 0; i < ideaArray.length; i++) {
+    if (ideaArray[i].star) {
+      ideaCardSection.insertAdjacentHTML(
+        'afterbegin',
+        `
+        <div class="card">
+          <header>
+            <button id="${ideaArray[i].id}" class="header-star star" type="button" name="button">
+              <img id="${ideaArray[i].id}" class="star-outline star" src="${starBoy(i)}" alt="">
+            </button>
+            <button id="${ideaArray[i].id}" class="header-close close" type="button" name="button">
+              <img id="${ideaArray[i].id}" class="close" src="./assets/menu-close.svg" alt="">
+            </button>
+          </header>
+          <section class="card-body">
+            <h4 class="card-title header-text">${ideaArray[i].title}</h4>
+            <p class="body-text">${ideaArray[i].body}</p>
+          </section>
+          <footer>
+            <button class="footer-button" type="button" name="button"><img class="comment-img" src="./assets/comment.svg" alt=""> Comment</button>
+          </footer>
+        </div>
+        `
+      )
+    }
+  }
+  showAllButton.classList.remove("hidden");
+  showStarredButton.classList.add("hidden");
+}
+
+function switchView() {
+  showAllButton.classList.add("hidden");
+  showStarredButton.classList.remove("hidden");
+  displayCard();
+}
+
+function inputSearch() {
```

Comment:
Glad we looked at the project overview again so that we have our search function updated to work in the input bar!